src/main/java/com/whereisagift/wishlist/WishlistController.java

```diff
@@ -34,19 +43,26 @@ public Iterable<Wishlist> wishlists(@AuthenticationPrincipal Long userId) {
     @MutationMapping
     @PreAuthorize("isAuthenticated()")
     @Transactional
-    public Wishlist createWishlist(@Argument WishlistInput wishlistInput, @AuthenticationPrincipal Long userId) {
-        List<Long> wishIds = wishlistInput.getWishIds();
+    public Wishlist createWishlist(
+            @Argument WishlistInput wishlistInput,
```

Comment:
```suggestion
         @Valid   @Argument WishlistInput wishlistInput,
```
Чтобы валидация инпута отработала